conda/wasp/meta.yaml

```diff
@@ -4,7 +4,7 @@
 #
 # As well as any directions pertaining to modifying those files.
 # ALSO: Follow the directions in scripts/tests/versioner_hashes.yaml
-{% set version = "2025.05.13" %}
+{% set version = "2025.07.07" %}
```

Comment:
Double-checking that this is the correct date you want to use (since it's dated July).